.nxxm/deps

```diff
@@ -1,10 +1,17 @@
 {
-   "daminetreg/js-bind" : { "@" : "v0.0.2" }
+   "nxxm/js-bind" : { "@" : "master" }
   , "aantron/better-enums" : { "@" : "0.11.1" }
 
   , "platform:gcc-7-cxx17" : [ "Boost::system", "OpenSSL::+SSL", "OpenSSL::+Crypto", "target::dl" ]
   , "platform:clang-cxx17" : [ "Boost::system", "OpenSSL::+SSL", "OpenSSL::+Crypto" ]
   , "platform:vs-15-2017-win64-cxx17" : [ "Boost::system", "OpenSSL::+SSL", "OpenSSL::+Crypto", "target::crypt32" ]
   , "platform:vs-16-2019-win64-cxx17" : [ "Boost::system", "OpenSSL::+SSL", "OpenSSL::+Crypto", "target::crypt32" ]
+  
+  , "platform:linux" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::dl" ]
+  , "platform:linux-cxx17" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::dl" ]
+  , "platform:macos" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit" ]
+  , "platform:macos-cxx17" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit" ]
+  , "platform:windows" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::crypt32" ]
+  , "platform:windows-cxx17" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::crypt32" ]
```

Comment:
Isn't it possible here to just write `"platform" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit"]`
`"platform:windows" : [ "target::crypt32"]`
?

It it doesn't then we need to fix it in nxxm (create an issue for it)

Reply:
So from ad hoc discussion I remember we have a bug there and it isn't possible.

Reply:
Issue created at https://github.com/nxxm/nxxm-src/issues/211